File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
//品牌服务层
app.service('brandService',function ($http) {
    //读取列表数据绑定到表单中
    this.findAll=function () {
        return $http.get('../brand/findAll.do');
    }
    this.findPage=function (page,size) {
        return $http.get('../brand/findPage.do?page='+page+'&size='+size);
    }
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
    this.add=function (entity) {
        return $http.post('../brand/add.do',entity);
    }
    this.update=function (entity) {
        return $http.post('../brand/update.do',entity);
    }
    this.dele=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }
    this.search=function (page,size,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&size='+size,searchEntity);
    }
    //下拉菜单
    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }
});

File: pinyougou-sellergoods-interdace/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

//品牌接口

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    //品牌分页
    public PageResult findPage(int pageNum,int pageSize);

    //返回下拉列表
    public List<Map> selectOptionList();

    //增加
    public void add(TbBrand brand);

    //根据id查询
    public TbBrand findOne(Long id);

    //修改
    public void update(TbBrand brand);

    //删除
    public void delete(Long[] ids);

    //品牌分页
    public PageResult findPage(TbBrand brand,int pageNum,int pageSize);


}
